src/main/java/com/mtb/demo/dto/ProductFilterDto.java

```diff
@@ -6,8 +6,23 @@
 import lombok.Getter;
 
 @Builder
-@Getter
```

Comment:
Why Getter removed?

Reply:
if removed then make a record class type for this dto

Reply:
returned @Getter, removed accidentally